index.js

```diff
@@ -336,7 +349,12 @@ controller.hears(/pay\s+([A-z|0-9]+)/i, 'direct_mention,direct_message,bot_messa
   var amount = invRecord.fields['Amount']
   var target = invRecord.fields['From']
   var note = `for invoice ${invRecord.id}`
-  var replyCallback = text => bot.replyInThread(message, text)
+  var replyCallback = text => {
+    bot.replyInThread(message, text)
+    if (typeof invoiceReplies[id] == "function") {
```

Comment:
isn't it supposed to be `invRecord.id`not `id` for the variable in the object.

Reply:
Those two are the same in this context. `id` is what's used to fetch the `invRecord`.

Reply:
oh great, didn't see that above